CompanyEmployees.Core.Domain/Exceptions/AttendanceAlreadyExistsException.cs

```diff
@@ -0,0 +1,9 @@
+namespace CompanyEmployees.Core.Domain.Exceptions;
+
+public sealed class AttendanceAlreadyExistsException : BadRequestException
+{
+    public AttendanceAlreadyExistsException(Guid employeeId)
+        : base($"Employee with ID {employeeId} has already checked in today.")
```

Comment:
Consider including the current date in the exception message to provide clearer context about when the user already checked in. This might help during debugging or API response reviews.

**Suggested change**
```suggestion
public AttendanceAlreadyExistsException(Guid employeeId)
    : base($"Employee with ID {employeeId} has already checked in on {DateTime.UtcNow:yyyy-MM-dd}.")
{
}
```